src/source/authentication.rst

```diff
@@ -235,7 +244,15 @@ After obtaining the authorization code at step 2 the client needs to make a
     Host: admarkt.demo.qa-mp.so
     Content-Type: application/x-www-form-urlencoded
 
-    grant_type=authorization_code&code=AUTH_CODE&client_id=YOUR_CLIENT_ID&client_secret=YOUR_CLIENT_SECRET&redirect_uri=https://yoursite.com/code
+    -d 'grant_type=authorization_code&code=AUTH_CODE&client_id=YOUR_CLIENT_ID&client_secret=YOUR_CLIENT_SECRET&rredirect_uri=https://yoursite.com/code' https://admarkt.marktplaats.nl/accounts/oauth/token
+.. warning::
+    Beware of the Content-type: build it as a `multipart/form-data`.
```

Comment:
the example above makes ut application/x-www-form-urlencoded, this is not in sync with this warning

Reply:
indeed we're not doing curl examples